features/bootstrap/FeatureContext.php

```diff
@@ -218,6 +218,39 @@ public function thereAreDummyPropertyObjects(int $nb)
         $this->manager->flush();
     }
 
+    /**
+     * @Given there are :nb dummy property objects with :nb2 groups
+     */
+    public function thereAreDummyPropertyObjectsWithGroups(int $nb, int $nb2)
+    {
+        for ($i = 1; $i <= $nb; ++$i) {
+            $dummyProperty = new DummyProperty();
+            $dummyGroup = new DummyGroup();
+
+            foreach (['foo', 'bar', 'baz'] as $property) {
+                $dummyProperty->$property = $dummyGroup->$property = ucfirst($property).' #'.$i;
+            }
+
+            $dummyProperty->group = $dummyGroup;
+
+            $this->manager->persist($dummyGroup);
+            for ($j = 1; $j <= $nb2; ++$j) {
+                $dummyGroup = new DummyGroup();
+
+                foreach (['foo', 'bar', 'baz'] as $property) {
+                    $dummyGroup->$property = ucfirst($property).' #'.$i.$j;
+                }
+
+                $dummyProperty->groups[] = $dummyGroup;
+
+                $this->manager->persist($dummyGroup);
+            }
```

Comment:
would put newline after bracked, and would remove empty lines between 244 and 246